tools/run_tests/performance/loadtest_config.py

```diff
@@ -94,34 +107,62 @@ def gen_run_indices(runs_per_test: int) -> Iterable[str]:
         yield prefix_fmt.format(i)
 
 
-def gen_loadtest_configs(base_config: yaml.YAMLObject,
-                         scenarios: Iterable[Mapping[str, Any]],
-                         loadtest_name_prefix: str,
-                         uniquifiers: Iterable[str],
-                         annotations: Mapping[str, str],
-                         runs_per_test: int = 1) -> Iterable[yaml.YAMLObject]:
-    """Generates LoadTest configurations as YAML objects."""
-    validate_annotations(annotations),
+def gen_loadtest_configs(
+        base_config: Mapping[str, Any],
+        scenario_name_regex: str,
+        language_config: scenario_config_exporter.LanguageConfig,
+        loadtest_name_prefix: str,
+        uniquifiers: Iterable[str],
```

Comment:
I've added quite a bit of documentation in the README file now, and renamed these items to "uniquifier_ elements" and so on. It is useful to have multiple uniquifier elements. For instance, when we want to run tests with the same configuration on two different node pools, we would need to give them different names. When we want to run them twice, we need to give htem different names, and so on. We could require the user to calculate and string together a uniquifier with multiple elements, but by naturally having multiple elements, I can call the config generator in a loop (nested loop, if it comes to that) and concatenate the whole thing at the end.

I have also added documentation on labels and annotations, these are standard kubernetes concepts. I am always adding the prefix as a label for instance, so I can run commands like this (prefix defaults to the user name):

```
$ kubectl delete loadtest --selector=prefix=paulosjca
```
Annotations are not indexed, so cannot be used in selectors, but they are still useful. I am always adding the scenario name and the uniquifier as annotations, since they are otherwise buried in the uuid and not human-readable. Additional annotations can be used as memos (just like comments).

Apart from the uses above, annotations can also be used by automation tools. For instance, if we are running tests on different node pools, or that otherwise don't compete for resources, I can tell a test runner to queue the based on an annotation.